webapp/packages/plugin-sql-editor/src/SqlEditor/SqlEditorController.tsx

```diff
@@ -81,7 +81,7 @@ export class SqlEditorController implements IInitializableController {
     indentWithTabs: true,
     smartIndent: true,
     autofocus: true,
-    lineWrapping: true,
+    lineWrapping: false,
```

Comment:
remove `lineWrapping`  from `SQLCodeEditorController` `COMMON_EDITOR_CONFIGURATION` const
also from this place